hazelcast-integration/kubernetes/samples/sidecar/config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hazelcast-configuration
+data:
+  hazelcast.xml: |-
```

Comment:
What about changing that to yaml after upgrading 3.12?

Reply:
Good idea, changed!